python_modules/dagster/dagster/_core/scheduler/instigation.py

```diff
@@ -377,6 +386,26 @@ def dynamic_partitions_request_results(
     ) -> Sequence[DynamicPartitionsRequestResult]:
         return self.tick_data.dynamic_partitions_request_results
 
+    @property
+    def asset_materialization_count(
```

Comment:
I'd call this `asset_materialization_request_count` or something.

It's now possible for sensor ticks to record AssetMaterialization events directly, so it would be good to be clear that this is not that.

Reply:
Separately, what will this return for a sensor tick that includes a run request that targets assets? I know it's inconsequential now because the frontend will never ask for it, but an answer of `0` could make it easy to write bugs in the future.

Reply:
wait how is it possible for sensor ticks to records asset materializations?

Reply:
@prha RunRequest has an asset_selection field now (I think from when AMP was using sensors more directly)